### DISCUSSION TITLE
Heslo na WiFi

### DISCUSSION THREAD

**Discussion by Petr6567** [community/community#119905] [Accessibility]
### Select Topic Area

Question

### Body

Dobrý den,sháním nějakou spolehlivou app která dokáže zjistit heslo na cizí WiFi.predem vám moc dekuji

**Comment by Akash1134** [comment]
Hi USERNAME,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as this one. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
